File: aoc2022/day1/part1.py


def main():
    with open('input.txt', 'r') as f:
        elves = f.read().split('\n\n')
    res = [list(map(int, elf.strip().split("\n"))) for elf in elves]
    current_max = 0
    for elf in res:
        if current_max <= sum(elf):
            current_max = sum(elf)
    print(current_max)


if __name__ == '__main__':
    main()

File: aoc2016/day2/part1.js
import fs, { read } from "fs";
import Keypad from "./keypad.js";

const readInput = (readFile) => {
  const file = fs.readFileSync(readFile, "utf-8").split("\n");
  return file;
};

let keypad = new Keypad();
console.log(keypad.pressAll(readInput("./input.txt")));
